contrib/platform/src/com/sun/jna/platform/win32/Guid.java

```diff
@@ -466,16 +468,14 @@ public CLSID(GUID guid) {
 
     /**
      * The Class REFIID.
-     *
-     * @author Tobias Wolf, wolf.tobias@gmx.net
      */
-    public class REFIID extends IID {
+    public class REFIID extends ByReference {
```

Comment:
That was my first thought and would align with the FAQ about this, but then this does not explain the behavior in callbacks - (please see the comment for the first commit, first paragraph)

The difference between my explicit use of ByReference and the suggested Structure/Structure.ByReference is the special casing the latter receives in com.sun.jna.CallbackReference (lines 511-516). My approach circumvents the autowrite.

The alternative would have been to mark REFIID not to autosync, but that looked worse.

Yes REFIID is typedeffed to IIID* (https://msdn.microsoft.com/en-us/library/cc237816.aspx)